README.md

```diff
@@ -6,6 +6,16 @@ Shared metadata for Exercism exercises.
 
 Please see the [contributing guide](./CONTRIBUTING.md).
 
+## Reviewing Guidelines
+
+Pull Requests to this repo require three approving reviews to be merged.
+
+- If you think that a PR should not be merged, either as is or in general, please leave a review **requesting changes**.
+  - This will block the PR from being merged until the change request has been addressed or overruled by an admin.
+  - Please also use the Request Changes button if you are unsure about how a concern of yours could be resolved.
+  - This ensures that your thoughts will be heard and can be addressed rather than being overruled by a majority.
```

Comment:
This list has items which are not of equal "list-indentness" in terms of information. I don't know how to solve, but right now it seems wrong 😅

Reply:
How about... Might be too much, but nested does seem a bit better for which points are "main" vs "follow-on"

-----

Pull Requests to this repo require three approving reviews to be merged.

- If you think that a PR should not be merged, either as is or in general, please leave a review **requesting changes**.
- Please also use the **Request Changes** button if you are unsure about how a concern of yours could be resolved.
    - This will block the PR from being merged until the change request has been addressed / overruled by an admin.

    - It ensures that your thoughts will be heard and can be addressed rather than being overruled by a majority.
    - When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping [@ErikSchierboom](https://github.com/ErikSchierboom/) to resolve the discussion.

Reply:
> When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping @ErikSchierboom to resolve the discussion.

This point is not linked to change requests, imo, so it should be on the higher level. Otherwise that change would be fine with me.